src/Adapter/Redis.php

```diff
@@ -36,31 +36,43 @@ public function setErrorCount(string $service = 'default', int $value = 0): void
     /**
      * @inheritdoc
      */
-    public function getErrorCount(string $service = 'default'): int
+    public function decrErrorCount(string $service = 'default', int $value = 1): void
     {
```

Comment:
I can't see a situation where I need to increment or decrement by a value different from 1.